### DISCUSSION TITLE
lsp-yaml to have Kubernetes auto-compilation.

### DISCUSSION THREAD

**Discussion by gnumonk** [emacs-lsp/lsp-mode#3878] [Q&A]
lsp-yaml to have Kubernetes auto-compilation the way vscode is doing. I have seen the variable which sets if ifle name has *-k8s.yaml or *-k8s.yml , it should enable the Kubernetes auto-compilation, but it is not ? Do let me know if I am missing some information or need to go through the man.

**Comment by yyoncho** [comment]
cc @kiennq
